.gitignore

```diff
@@ -16,3 +16,4 @@ demo.md
 generators/*.gz
 /csvs/
 .unotes/*
+!workshop/5-Conversational/RetailDemoStore_Lex.zip
```

Comment:
why this zip in the gitignore? need it for import in Lex console, right?

Reply:
Yes, the zip file is needed to import the chatbot config in Lex. It's very small. Just contains a JSON file.